diff -Naur libjpeg-turbo-2.1.5.1.old/CMakeLists.txt libjpeg-turbo-2.1.5.1/CMakeLists.txt
--- libjpeg-turbo-2.1.5.1.old/CMakeLists.txt	2026-05-29 10:59:04.710952309 +0200
+++ libjpeg-turbo-2.1.5.1/CMakeLists.txt	2026-05-29 11:01:27.965426644 +0200
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.5...3.10)
 # When using CMake 3.4 and later, don't export symbols from executables unless
 # the CMAKE_ENABLE_EXPORTS variable is set.
 if(POLICY CMP0065)
